cmd/admin-heal-ops.go

```diff
@@ -543,24 +541,59 @@ func (h *healSequence) healConfig() error {
 		return errServerNotInitialized
 	}
 
-	configFile := path.Join(minioConfigPrefix, minioConfigFile)
-	configBackupFile := path.Join(minioConfigPrefix, minioConfigBackupFile)
-	for _, cfg := range []string{configFile, configBackupFile} {
-		res, err := objectAPI.HealObject(h.ctx, minioMetaBucket, cfg, h.settings.DryRun)
+	marker := ""
+	isTruncated := true
+	for isTruncated {
+		if globalHTTPServer != nil {
+			// Wait at max 1 minute for an inprogress request
+			// before proceeding to heal
+			waitCount := 60
+			// Any requests in progress, delay the heal.
+			for globalHTTPServer.GetRequestCount() > 0 && waitCount > 0 {
+				waitCount--
+				time.Sleep(1 * time.Second)
+			}
+		}
+
+		// Heal numCPU * nodes objects at a time.
```

Comment:
Not able to understand how this comment is applicable.

Reply:
Remnant of a different code.